src/sr_od/app/sim_uni/operations/bless/bless_utils.py

```
@@ -95,10 +112,14 @@ def get_bless_by_priority(bless_list: List[SimUniBless], config: Optional[SimUni
         for bless_level in SimUniBlessLevel:
             for idx, opt_bless in enumerate(bless_list):
-                if opt_bless.level == bless_level:
-                    if idx_priority[idx] == 99:
-                        idx_priority[idx] = cnt
-                        cnt += 1
+                try:
+                    if opt_bless.level == bless_level:
+                        if idx_priority[idx] == 99:
+                            idx_priority[idx] = cnt
+                            cnt += 1
+                except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)